synchronous-agent-invocation/requirements.txt

```diff
@@ -1,5 +1,5 @@
 requests==2.31.0
-cortex-python==6.0.3
+cortex-python==6.3.1
```

Comment:
there are breaking api changes with this version. hope the example code has been updated

Reply:
looks like this example doesnt use any of the updated classes/methods

Reply:
Yeah, this example behaved same as it was behaving for 6.0.3